### DISCUSSION TITLE
Bandwidth of odf

### DISCUSSION THREAD

**Discussion by julesbaton** [mtex-toolbox/mtex#851] [Ask Anything] [answered]
Hi everyone,
I am using the calcODF function to determine an ODF from pole figures : 
odf = calcODF(pf,'halfwidth',5*degree,'resolution',5*degree)

I'm not sure what the ODF "bandwidth" property is. Does this correspond to the maximum order of the Fourier expansion of the linear combination of unimodal ODFs (describing the whole ODF) or did I misunderstand ?

Thank you for your reply,
Jules

**Comment by ralfHielscher** [comment, marked as answer]
In the line of code there is not ```bandwidth``` property :smile: Apart from the bandwidth is exactly what you said. The halfwidth is explained here: https://mtex-toolbox.github.io/DensityEstimation.html

Ralf.

**Reply by julesbaton** [reply]
Thank you for the quick reply.
You are right, I should have said when I look at the properties of the calculated ODF, there is the bandwidth property to be less ambiguous.
Thanks for the link and for your help.
Jules
